templates/VALIDATION.md

```diff
@@ -147,8 +147,10 @@ kind: Template
 ### Optional Keys
 
 * `valid`: the rule must be *ignored* if the jsonpath given as value doesn't exist.
-*PLEASE NOTE* that even if values of this key are required to be JSONPaths, you still need to use the `jsonpath::` prefix
-as explained above.
+  Some of the fields of the template have default values so they always exist, and setting this path to one of these fields
+  has no effect.
```

Comment:
I know this can be tiresome, but can you please add some examples to such values? or at least values we already know of

Reply:
All fields that are not defined as pointers, slices or maps in `VirtualMachineInstanceSpec` have default values. There are a lot of them. I could list a few, maybe it will be useful.

Reply:
If you could list few of the important ones (like cpu) it would be great

Reply:
Ok, I have added some fields that are probably important.